python/pyarrow/tests/test_convert_builtin.py

```diff
@@ -2185,7 +2185,11 @@ def test_auto_chunking_list_like():
     assert arr.num_chunks == 2
     assert len(arr.chunk(0)) == 7
     assert len(arr.chunk(1)) == 1
-    assert arr.chunk(1)[0].as_py() == list(item)
```

Comment:
Converting to python object takes a lot of time, so assert on arrow objects instead.